File: Leetcode/Java/Easy/26RemoveDuplicatesfromSortedArray.java
import java.util.ArrayList;

class Solution {
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer> unique_numbers = new ArrayList<>();
        for(int i : nums){
            if (!unique_numbers.contains(i)) {
                unique_numbers.add(i);
            }
        }

        for(int i =0; i <unique_numbers.size(); i++){
            nums[i] = unique_numbers.get(i);
        }   

        return unique_numbers.size();

    }
}
//Accepted answer

File: Leetcode/Rust/Easy/26RemoveDuplicatesfromSortedArray.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        let mut unique_numbers: Vec<i32> = Vec::new();
        
        
        for &value in nums.iter() {
            if !unique_numbers.contains(&value) {
                unique_numbers.push(value);
            }
        }

        for i in 0..unique_numbers.len() {
            nums[i] = unique_numbers[i];
        }

        unique_numbers.len() as i32
    }
}
